.github/workflows/cron-translations.yml

```diff
@@ -0,0 +1,47 @@
+on:
+  cron:
+    - cron: "19 58 * * *" # every day at 7:58 PM UTC
+
+name: Pull translations from Crowdin
+
+jobs:
+  crowdin_pull:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: download from crowdin
+        uses: docker://jonasfranz/crowdin
+        env:
+          CROWDIN_KEY: ${{ secrets.CROWDIN_KEY }}
+          PLUGIN_DOWNLOAD: true
+          PLUGIN_EXPORT_DIR: options/locale/
+          PLUGIN_IGNORE_BRANCH: true
+          PLUGIN_PROJECT_IDENTIFIER: gitea
+      - name: update locales
+        run: ./build/update-locales.sh
+      - name: push translations to repo
+        uses: appleboy/git-push-action@main
```

Comment:
change to `appleboy/git-push-action@v0.0.2` stable version.

Reply:
thanks. updated :)